src/controllers/connection/helpers.ts

```diff
@@ -15,7 +16,7 @@ export const checkDb = (rows: ConnectionRow[], initialPinAttemptsRemaining: numb
     }
   }
   if (connectionCheck.pin_tries_remaining_count === 0) {
-    logger.debug('Maximum number of pin attempts has been reached.')
+    logger.debug('Maximum number of pin attempts has been reached. [%s]', connectionCheck.pin_tries_remaining_count)
```

Comment:
Would it be useful to rewrite this as:
`logger.debug('Maximum number of pin attempts has been reached. Remaining attempts: [%s]', connectionCheck.pin_tries_remaining_count)`

Just so we know what that number means

Reply:
updated.